Vagrantfile

```diff
@@ -16,6 +16,8 @@ Vagrant.configure("2") do |config|
 
   config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
   
+  config.vm.provision "shell", path: "setup.sh", privileged: false
```

Comment:
The PR description and commit message mention `bootstrap.sh` while you actually use `setup.sh`. I don't think it matters which one you use, but make sure the commit message, PR description and actual changes match.

Reply:
please rename the PR accordingly as well

Reply:
@MeitarRizner please address this comment.